conf/mesos-site.xml

```diff
@@ -19,54 +19,84 @@
 <configuration>
   <property>
     <name>mesos.hdfs.data.dir</name>
+    <description>The primary data directory in HDFS</description>
     <value>/tmp/hdfs/data</value>
   </property>
   
   <property>
     <name>mesos.hdfs.secondary.data.dir</name>
+    <description>The secondary data directory in HDFS</description>
     <value>/var/run/hadoop-hdfs</value>
   </property>
-  
+
+  <property>
+    <name>mesos.hdfs.native-hadoop-binaries</name>
+    <description>Mark true if you have hadoop pre-installed on your host machines (otherwise it will be distributed by the scheduler)</description>
+    <value>false</value>
+  </property>
+
   <property>
     <name>mesos.hdfs.framework.mnt.path</name>
+    <description>Mount location (if above config is marked false)</description>
     <value>/opt/mesosphere</value>
   </property>
 
   <property>
     <name>mesos.hdfs.state.zk</name>
     <value>localhost:2181</value>
+    <description>Hostname and port of zookeeper node location</description>
```

Comment:
That's in the quorum field. This is used by the framework to write into ZK, so you only need one. The quorum is what HDFS uses for its own zookeeper handling (and i'm testing right now whether it actually requires multiple unique DNS entries).